pkg/pkg.bzl

```diff
@@ -287,10 +295,14 @@ def pkg_tar(**kwargs):
                       "This attribute was renamed to `srcs`. " +
                       "Consider renaming it in your BUILD file.")
                 kwargs["srcs"] = kwargs.pop("files")
-    _real_pkg_tar(**kwargs)
+    extension = kwargs.get("extension") or "tar"
+    pkg_tar_impl(
+        out = kwargs["name"] + "." + extension,
+        **kwargs
+    )
 
 # A rule for creating a deb file, see README.md
-pkg_deb = rule(
+pkg_deb_impl = rule(
```

Comment:
@aiuto as we have public `pkg_deb`, should this one be exported too?

Reply:
Sigh.  No. 
The problem is that with the removal of `outputs` you can no longer default an output file name. So you need a macro. You can not use the real implementation directly or you won't set all the mandatory outputs.